kernel/runtime/memory/runtime/umem.cc

```diff
@@ -0,0 +1,27 @@
+#include "runtime/SequentialHeap.hh"
+
+
+mythos::SequentialHeap<uintptr_t> heap;
```

Comment:
This should have a matching header file in order to access the heap instance directly for addRange(). The implementation of the heap should be changeable at compile time. This can be solved by separating the new,delete,malloc,free implementations into a separate module. The cc file includes the umem.hh to access the heap instance without caring about its implementation type.